### DISCUSSION TITLE
Discord mods

### DISCUSSION THREAD

**Discussion by Silentlurker69** [skyline-emu/skyline#1701] [General]
I just saw some stuff that is not okay on the discord. You guys need to better vet your moderators if you want this community to grow. Some aggressive guy joined the discord ( as per usual), and one moderator, Rob, among others was equally as toxic and said things that could potentially be triggering to some people. Very unprofessional of a community moderator. I wasn't sure where to go with this, but this is definitely not okay. This needs to be looked into!

**Comment by dima-xd** [comment]
Chill. No democracy. Mods do what they want

**Comment by Swizzlefowl** [comment]
Doesn't follow the server rules, uses an alt to call moderator toxic and uses gay as an insult. Wonders why he got banned.

**Comment by PixelyIon** [comment]
This is not exactly the right forum to complain about moderators, the Skyline repository is specifically for discussion that relates to the application in itself but you've said as much so I don't take much of an issue with this instance of it. 

After reviewing the situation internally we've determined that the toxicity complained about here was related to not following Rule 1 of the server: "Asking, linking or helping someone download ROMs or other copyrighted content in any way is **NOT** allowed. **Piracy is completely prohibited**". We've determined that the helper was in the right to prohibit breaking sever rules, this rule is in place to prevent breaking the Discord ToS and copyright law which we do need to follow to protect ourselves.

**Comment by Silentlurker69** [comment]
There was no helping or asking of downloading of illegal content going on,
y'all are just some powertrippin bozos. What else could one expect of
discord mods i suppose. Eat my shorts tankie

**Comment by PixelyIon** [comment]
We aim to be reasonable and fair to everyone but that relation is expected to be mutual, I don't believe that the hostility in your response is warranted. Regardless, I will look into the situation further and determine if what you said was the case (and disparage helpers if they were in the wrong) but due to your lack of following etiquette here I will be blocking you from the repository.

**Comment by Silentlurker69** [comment]
Do you just choose to just ignore what your mods say? Hilarious.
"uNwArRaNtEd" maybe if you took the time to see what your abusive mods did
you would understand. There was nothing of piracy but your mod went out of
the way to try to accuse people of it as an excuse to ban. "Mods do what
they want" is a reply i got from the mods. Do you really think this power
hungry response is okay when trying to cultivate a community?

**Reply by bylaws** [reply]
Could you provide more information to me or mark on DMs, rather than a public forum.

**Reply by mateoalfaro** [reply]
The "mods do what they want" reply wasn't even from a moderator

**Reply by PixelyIon** [reply]
Correct, that comment was not from any helper/moderator, it's from someone who's banned from the server. Regardless, I feel there's little room for further discussion of this issue here. If @Silentlurker69 wants to report a moderator's actions they can with an apt description of what happened to either of us in DMs (`◱ Mark#4467` or `ByLaws#8238`) and so far they've been fairly unreliable and inflammatory. I will be locking this issue as stated prior, this is not the right forum for these sorts of issues.
